CMakeLists.txt

```diff
@@ -368,6 +368,8 @@ ELSE()
   MESSAGE("-- World Builder version ${WorldBuilderVersion} found.")
   SET(ASPECT_WITH_WORLD_BUILDER ON CACHE BOOL "If ON the World Builder is compiled togheter with ASPECT.")
 
+  configure_file("${CMAKE_SOURCE_DIR}/contrib/WorldBuilder/source/config.cc.in" "${CMAKE_SOURCE_DIR}/contrib/WorldBuilder/source/config.cc" @ONLY)
```

Comment:
how does this work? the .in file has a bunch of version strings that are not defined in here or does it?

Reply:
Yes, you where correct that it was not complete. Sorry for that. This was a quick fix to make it compile for someone who was planning to use aspect and the world builder. I now fully tested it and I managed to get it working on a complex 3d example.